core/os/device/device.proto

```diff
@@ -56,6 +56,21 @@ enum OSKind {
   Stadia = 5;
 }
 
+// Incompatibility list possible reasons why a device is not compatible to
+// replay a given trace.
+enum Incompatibility {
```

Comment:
I'm not a big fan of the double-negative. I'd prefer calling this `ReplayCompatability` and possibly prefixing (all but the first) enums with a negative (i.e. kind of like `VkResult`). However, I see why'd you choose this, so, up to you.

Reply:
Renamed to ReplayCompatibility